src/storage/executor/command.rs

```diff
@@ -573,6 +629,24 @@ impl Command {
                 let result = vec![prefix];
                 return result;
             }
+            Command::RemoveGroupings { groupings } => {
+                let prefix = CommandPrefix::RemoveGroupings as u8;
+                let groupings_bytes: Vec<Vec<u8>> = groupings
+                    .iter()
+                    .map(|grouping| grouping.marshal())
+                    .collect();
+
+                let mut result = vec![prefix];
+
+                let num_of_groupings = groupings.len();
```

Comment:
Do we need this information? Doesn't each `grouping` contain its own length?

Reply:
done